### PR TITLE
[Feature] Import: add or replace

### PR DESCRIPTION
**Pull request by clfer**
New functionality:
Allow to choose between replacing existing rules or adding to existing rules when importing a file

### CODE REVIEW

**Comment by clfer** [comment]
import-add_or_replace-demo

File: src/js/options/models/tab_modifier.model.js [hunk 1]
```diff
@@ -28,16 +28,19 @@ app.factory('TabModifier', ['Rule', function (Rule) {
...
         var self = this;
         
         if (data.settings !== undefined) {
             this.settings = data.settings;
         }
-        
-        this.deleteRules();
-        
+
+        if (replaceExistingRules) {
```
Comment (sylouuu) [inline src/js/options/models/tab_modifier.model.js]:
```js
// In this project, snake_case is used for variables
// Please rename your variables
// Also use strict comparison
if (replace_existing_rules === true)
```

File: src/js/options/models/tab_modifier.model.js [hunk 2]
```diff
@@ -64,9 +67,10 @@ app.factory('TabModifier', ['Rule', function (Rule) {
             return false;
         }
     };
-    
-    TabModifier.prototype.import = function (json) {
-        this.build(JSON.parse(json));
+
+    TabModifier.prototype.import = function (json, replaceExistingRules) {
+        replaceExistingRules = typeof replaceExistingRules !== 'undefined' ? replaceExistingRules : true;
+        this.build(JSON.parse(json), replaceExistingRules);
```
Comment (sylouuu) [inline src/js/options/models/tab_modifier.model.js]: Please insert a line break after assignment

**Comment by sylouuu** [comment]
Thank you @clfer :)

**Comment by sylouuu** [comment]
Thanks for contributing!

**Comment by sylouuu** [comment]
Released in 0.22.0!

**Pull request merged**